app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -644,6 +645,21 @@ private Feed loadData() {
         return feed;
     }
 
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onKeyUp(KeyEvent event) {
```

Comment:
Just for precaution, I would check if `isAdded() && isVisible()`. If the fragment is not shown, no key codes should ever be handled, even if it somehow is still registered to the events.

Reply:
Could you also add the same method to the "all episodes", "playback history" and "queue" screens, please?